shell.nix

```diff
@@ -16,4 +16,4 @@
 ) {
   src =  ./.;
 }).shellNix
-# nix-direnv cache busting line: sha256-av4kr09rjNRmag94ziNjJuI/cg8b8lAD3Tk24t/ezH4=
+# nix-direnv cache busting line: sha256-cQRbR7e1BYgd49AfuuPHuKhjCG2tn9DenRlTqQyrHv8=
```

Comment:
Blocking: Remove the Nix changes, they are irrelevant to this patch.

If you need to make changes to build on your own system, do not `git add` them to your commits.